CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Adding `behaviors` section to the menu, under the components @kolaps33 ([#119](https://github.com/stardust-ui/react/pull/119))
 - Add `avatar` prop to `Chat.Message` subcomponent @Bugaa92 ([#159](https://github.com/stardust-ui/react/pull/159))
 - add `iconOnly` prop to `Button` @mnajdova ([#182](https://github.com/stardust-ui/react/pull/182))
+- Create StatusIndicator as a separate component @musingh1 ([#168](https://github.com/stardust-ui/react/pull/168))
```

Comment:
for the sake of consistency with other CHANGELOG entries, please, brace `StardustIndicator` with back ticks (`) - as it is done for all types/properties/methods. Thanks!